File: CuriousDrive/CuriousDriveService/Models/busBase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CuriousDriveService.Models
{
    public abstract class busBase
    {
        public List<busMessage> ilstbusMessages { get; set; }
        public bool isOwner { get; set; }

        public busBase()
        {
            ilstbusMessages = new List<busMessage>();
        }

        public void AddMessage(int aintMessageId)
        {
            ilstbusMessages.Add(new AdminService().GetMessage(aintMessageId));
        }
    }
}

File: CuriousDrive/CuriousDriveWebService/src/main/java/com/org/panthers/framework/busBase.java
package com.org.panthers.framework;

import java.util.ArrayList;
import java.util.List;


public abstract class busBase {

	public List<busMessage> ilstbusMessages;
	public boolean isOwner;	
		
	public busBase(){
		ilstbusMessages = new ArrayList<busMessage>();
	}
	
	public void addMessage(int aintMessageId){
		
		busMessage lbusMessage = new busMessage();
		lbusMessage = new busMessage();
		lbusMessage.idoMessage.setMessageId(aintMessageId);
		lbusMessage.idoMessage = lbusMessage.idoMessage.get();
		ilstbusMessages.add(lbusMessage);
	}

}
